### PR TITLE
Change package License to Apache 2.0

### PR DESCRIPTION
**Pull request by astj**
Discussed on #360.
https://github.com/mackerelio/mackerel-agent/pull/360#discussion_r107351282
https://github.com/mackerelio/mackerel-agent/pull/360#discussion_r107351307

Since mackerel-agent itself is published under Apache License 2.0 and we don't add copyrighted stuff to package releases, we can release the packages under Apache License 2.0 too.
Currently they're released under Commercial License, but it's maybe because at first we don't publish mackerel-agent's source codes.

And also I updated copyright terms in the packages. We, Hatena should be referred as `Hatena Co., Ltd`.

### CODE REVIEW

File: packaging/rpm/mackerel-agent-systemd.spec [hunk 1]
```diff
@@ -6,14 +6,14 @@
 Name:      mackerel-agent
 Version:   %{_version}
 Release:   1%{?dist}
-License:   Commercial
+License:   ASL 2.0
```
Comment (astj) [inline packaging/rpm/mackerel-agent-systemd.spec]: https://fedoraproject.org/wiki/Packaging:LicensingGuidelines

> The License: field must be filled with the appropriate license Short License identifier(s) from the "Good License" tables on the Fedora Licensing page.

And Short License Identifier for Apache License 2.0 is `ASL 2.0`, according to the table.
https://fedoraproject.org/wiki/Licensing:Main

File: packaging/deb-systemd/debian/copyright [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 
 Copyright:
 
-    <Copyright (C) 2014 Hatena>
+    <Copyright (C) 2017 Hatena Co., Ltd.>
 
 License:
 
-    Commercial
+    Apache-2.0
```
Comment (astj) [inline packaging/deb-systemd/debian/copyright]: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/#license-specification

**Comment by itchyny** [comment]
LGTM

**Comment by astj** [comment]
Thanks!

**Pull request merged**